src/Tribe/Views/V2/View.php

```diff
@@ -247,7 +246,7 @@ public static function make_for_rest( \WP_REST_Request $request ) {
 			$params['prev_url'] = untrailingslashit( $params['prev_url'] );
 		}
 
-		$slug       = Arr::get( $params, 'view', false );
+		$slug       = Arr::get( $params, View::query_var(), false );
```

Comment:
:no_entry_sign: Must use "self::" for local static member reference

<sub>hash: aadaaa1725513afcbf412bad54f5bc0f</sub>